### ISSUE TITLE
The page will still stop on blank after sign-in

### ISSUE THREAD

**Issue by hang-ren** [DevExpress/testcafe#1922]
### Are you requesting a feature or reporting a bug?
Bug

### What is the current behavior?

Sign-in page will stay on blank

### What is the expected behavior?

Sign-in successfully.

### How would you reproduce the current behavior (if this is a bug)?
The case will be successful if run it manually, but failed on automation with testcafe

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:
http://ezhome-alpha-3d.shejijia.com/?brand=ezhome
Test code

testLogin.js
```js
import { Selector } from 'testcafe';
import loginFunc from '../general/login';

const loginFunction = new loginFunc();
const loginFromFirstPage = Selector('button.btn_loginbtn');
const closeWelcomeImg = Selector('img.closewelcome');
const loginLink = Selector('a.loginpp');
const loginIframe = Selector('#loginFrame');
const usrnameInput = Selector('#username');
const pwdInput = Selector('#password');
const loginButton = Selector("a.sso-submit");
const usernameText = Selector('span.nametext');

fixture `Login`
    .page `http://ezhome-alpha-3d.shejijia.com/?brand=ezhome`;

test('login_positive', async t => {
     /* Test Login */

    await loginFunction.login();
});
```

login.js
```js
import { t } from 'testcafe';

var loginCss = require('../general/elements.js');

export default class loginFunc {
    constructor () {
		this.continueBtn = loginCss.continueBtn;
		this.closeWelcomeImg = loginCss.closeWelcomeImg;
		this.loginLink = loginCss.loginLink;
		this.loginIframe = loginCss.loginIframe;
		this.usrnameInput = loginCss.usrnameInput;
		this.pwdInput = loginCss.pwdInput;
		this.loginButton = loginCss.loginButton;
		this.usernameText = loginCss.usernameText;        		
    }
	
	async login () {
        await t
            .maximizeWindow()
            .click(this.continueBtn)
            .click(this.closeWelcomeImg)
            .wait(2000)
            .click(this.loginLink)
            .switchToIframe(this.loginIframe)
            .wait(2000)
            .typeText(this.usrnameInput, loginCss.username)
            .typeText(this.pwdInput, loginCss.password)
            .click(this.loginButton)
            .switchToMainWindow()
            .wait(2000)
            .expect(this.usernameText.textContent).contains('爱飞SH', 'Actual username is not as expected.')
    }
}

```

elements.js
```js
import { Selector } from 'testcafe';

var username = '15502108088';
var password = '123456';
const continueBtn = Selector('button.btn_continuebtn');
const closeWelcomeImg = Selector('img.closewelcome');
const loginLink = Selector('a.loginpp');
const loginIframe = Selector('#loginFrame');
const usrnameInput = Selector('#username');
const pwdInput = Selector('#password');
const loginButton = Selector("a.sso-submit");
const usernameText = Selector('span.nametext');

exports.continueBtn = continueBtn;
exports.closeWelcomeImg = closeWelcomeImg;
exports.loginLink = loginLink;
exports.loginIframe = loginIframe;
exports.usrnameInput = usrnameInput;
exports.pwdInput = pwdInput;
exports.loginButton = loginButton;
exports.usernameText = usernameText;
exports.username = username;
exports.password = password;
```

### Specify your

* operating system: Windows 7
* testcafe version:0.18.2
* node.js version:v6.11.4

**Comment by hang-ren** [comment]
![screenshot](https://user-images.githubusercontent.com/29770618/32313287-0cc98686-bfdd-11e7-8a92-5c20ef405b27.JPG)

**Comment by AlexanderMoskovkin** [comment]
Hi @hang-ren, thanks for the report. I've reproduced the issue. 

Reproduced in `testcafe-hammerhead` playground /cc @miherlosev @LavrovArtem

**Comment by hang-ren** [comment]
Thanks Alex.
By the way, when will Sprint10 be released? I saw you added tag for the fixing.

**Comment by hang-ren** [comment]
@AlexanderMoskovkin Thanks!
By the way, when will Sprint#10 be released. I saw you added tag for the fixing.

**Comment by AlexanderMoskovkin** [comment]
@hang-ren We can provide you with a build with the fix once it's ready. The fix is in the [pull request](https://github.com/DevExpress/testcafe-hammerhead/pull/1365) already.

**Comment by hang-ren** [comment]
@AlexanderMoskovkin ok, thanks for the quick response!

**Comment by AlexanderMoskovkin** [comment]
The fix is published in `testcafe@0.18.4`
